### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by Liyang131313** [eugenelet/kMeans_Hardware#2]
Hello, is this the hardware implementation of the k-means algorithm? How can I use it?

**Comment by eugenelet** [comment]
@Liyang131313 Yes. It requires a synthesis tool. All files for RTL simulation is in 01_RTL and for gate level simulation you can refer to 03_SYN.
